src/components/Footer.tsx

```diff
@@ -41,14 +41,13 @@ const Footer = () => {
                         <Grid item xs={12} sm={3} lineHeight={2}>
                             <Box sx={styles.bottomLine}>CONTACT</Box>
                             <Box>
-                                <Link
-                                    href="/contact"
+                                <RouteLink
+                                    to="/contact"
                                     target="_blank"
```

Comment:
Personally, I would remove the `target="_blank"` attribute so it won't open a new tab when clicking on the contact page.

Reply:
Thanks for this suggestion, I agree with it too. It has been modified.